src/datatypes/point.h

```diff
@@ -14,3 +14,6 @@ float Point_lat(SIValue point);
 // returns longitude of given point
 float Point_lon(SIValue point);
 
+// return coordinate (latitude or longitude) of given point
```

Comment:
```
// returns a coordinate (latitude or longitude) of a given point
```

Reply:
Done in [666387d](https://github.com/RedisGraph/RedisGraph/pull/2524/commits/666387d61e00cc67bae2abd7f23a1fd8803902b6)